packages/recommend-react/src/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './useRecommendations';
 export * from './useRelatedProducts';
 export * from './useTrendingFacets';
 export * from './useTrendingItems';
+export * from './RecommendContext';
```

Comment:
This export means that `RecommendContext` and `useRecommendContext` are also exposed to users, but they should probably remain internals.

Having an explicit `Recommend.tsx` file (see my suggestion above renaming `RecommendProvider` to `Recommend`) would make the entry point easier to find for contributors, and likely reduce the risk of exporting more than needed.